docs/guides/persistence.mdx

```diff
@@ -73,6 +73,68 @@ const atomWithAsyncStorage = (key, initialValue) => {
 }
 ```
 
+### Usage in Async actions
+
+When using an asynchronous storage, the reading of the created atoms are asynchronous too. That comes with some limitations. Before `useAtom` has been called at least once, the value is not set.
+This can cause an issue, for example, if you want to retrieve the value of your stored atom in an async action (write-only atom).
+
+Let's use the [atomWithStorage function in the utils bundle](../api/utils.mdx#atom-with-storage) for the examples, so that we have more options available.
+
+```typescript
```

Comment:
If you are using `any` anyway, can we just use`js` here and remove typing?

Reply:
@dai-shi Yep sure, typing is not a concern here, the example is already complicated. enough